packages/frontend-dynamic-feature-loader/config.d.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /**
+   * @visibility frontend
+   */
+  dynamicPlugins?: {};
```

Comment:
another common pattern is `true` as a boolean for the intent of "enable with defaults" , but maybe there's going to be many more fields here later on so it doesn't make sensed to introduce that?

Reply:
There are other already other mandatory fields here on the backend side.
The main addition here is that we add the frontend visibility.

Reply:
There are already other mandatory fields here on the backend side.
The main addition here is that we add the frontend visibility.